region/maridia/inner-green/East Pants Room.json

```diff
@@ -410,6 +410,36 @@
         {"or": [
           "canTrickyJump",
           {"enemyDamage": {"enemy": "Puyo", "type": "contact", "hits": 2}}
+        ]},
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"enemyDamage": {"enemy": "Puyo", "type": "contact", "hits": 1}},
+          "Wave",
+          "canPreciseGrapple"
+        ]}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Freeze the Puyo at standing-shot height and then jump on top of it with a spinjump before morphing, unmorphing, and using X-Ray to standup and clip.",
+        "Standing on a side of the room and shooting forward will freeze the Puyo at the correct height."
+      ]
+    },
+    {
+      "link": [3, 2],
+      "name": "Suitless Puyo Clip with Morph and X-Ray",
+      "requires": [
+        {"notable": "Puyo Clip"},
+        "h_canXRayMorphIceClip",
+        "canTrickyJump",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"enemyDamage": {"enemy": "Puyo", "type": "contact", "hits": 2}}
+        ]},
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"enemyDamage": {"enemy": "Puyo", "type": "contact", "hits": 1}},
+          "Wave",
+          "canPreciseGrapple"
```

Comment:
Took me a while to understand this last OR. It's for the Puyo by the door, right?

Reply:
yes, it will try to stand above you while you are clipping.